content/shmem_init.tex

```diff
@@ -33,16 +34,18 @@
     None.
 }
 
+\begin{DeprecateBlock}
 \apinotes{
     As of \openshmem[1.2], the use of \FUNC{start\_pes} has been
     deprecated and calls to it should be replaced with calls to \FUNC{shmem\_init}.
     While support for \FUNC{start\_pes} is still required in \openshmem libraries,
     users are encouraged to use \FUNC{shmem\_init}. An important difference between
-    \FUNC{shmem\_init} and \FUNC{start\_pes} is that multiple calls to
-    \FUNC{shmem\_init} within a program results in undefined behavior, while in the
-    case of \FUNC{start\_pes}, any subsequent calls to \FUNC{start\_pes} after the
+    \FUNC{shmem\_init} and \FUNC{start\_pes} is that every call to
+    \FUNC{shmem\_init} within a program must be matched with a call to \FUNC{shmem\_finalize}.
+    In the case of \FUNC{start\_pes}, any subsequent calls to \FUNC{start\_pes} after the
     first one results in a no-op.
```

Comment:
```suggestion
```

Reply:
@wrrobin Please confirm that this start_pes behavior is documented in that section already.

Reply:
Confirmed that this is documented in the start_pes API notes.